frontend/model/state.js

```diff
@@ -485,12 +485,14 @@ const getters = {
     return profiles
   },
   ourContactProfilesById (state, getters) {
```

Comment:
this getter is named **our**ContactProfilesById, but the most important filter logic is missing inside it, which is only filtering the users that are members of the currently active group. So added the logic in as the fix.